test/new-e2e/tests/npm/ec2_1host_test.go

```diff
@@ -97,176 +86,106 @@ func TestEC2VMSuite(t *testing.T) {
-		})
-
-		hostPayloads := cnx.GetPayloadsByName(targetHostnameNetID)
-		lenHostPayloads := len(hostPayloads)
-		if !assert.Equalf(c, len(hostPayloads[lenHostPayloads-2].Connections), 600, "can't found enough connections 600+") {
-			return
-		}
-
-		cnx600PayloadTime := hostPayloads[lenHostPayloads-2].GetCollectedTime()
-		latestPayloadTime := hostPayloads[lenHostPayloads-1].GetCollectedTime()
-
-		dt := latestPayloadTime.Sub(cnx600PayloadTime).Seconds()
-		t.Logf("hostname+networkID %v diff time %f seconds", targetHostnameNetID, dt)
-
-		assert.Greater(t, 0.1, dt, "delta between collection is higher than 100ms")
-	}, 90*time.Second, time.Second, "not enough connections received")
+		test1HostFakeIntakeNPM600cnxBucket(&v.BaseSuite, v.Env().FakeIntake)
+	})
 }
 
 // TestFakeIntakeNPM_TCP_UDP_DNS validate we received tcp, udp, and DNS connections
 // with some basic checks, like IPs/Ports present, DNS query has been captured, ...
 func (v *ec2VMSuite) TestFakeIntakeNPM_TCP_UDP_DNS() {
-	t := v.T()
+	vt := v.T()
 	testURL := "http://" + v.Env().HTTPBinHost.Address + "/"
-
-	// generate connections
-	v.Env().RemoteHost.MustExecute("curl " + testURL)
-	v.Env().RemoteHost.MustExecute("dig @8.8.8.8 www.google.ch")
-
-	v.EventuallyWithT(func(c *assert.CollectT) {
-
-		cnx, err := v.Env().FakeIntake.Client().GetConnections()
-		assert.NoError(c, err, "GetConnections() errors")
-		if !assert.NotEmpty(c, cnx.GetNames(), "no connections yet") {
-			return
-		}
-
-		foundDNS := false
-		cnx.ForeachConnection(func(c *agentmodel.Connection, cc *agentmodel.CollectorConnections, hostname string) {
-			if len(c.DnsStatsByDomainOffsetByQueryType) > 0 {
-				foundDNS = true
-			}
-		})
-		if !assert.True(c, foundDNS, "DNS not found") {
-			return
-		}
-
-		type countCnx struct {
-			hit int
-			TCP int
-			UDP int
-		}
-		countConnections := make(map[string]*countCnx)
-
-		helperCleanup(t)
-		cnx.ForeachConnection(func(c *agentmodel.Connection, cc *agentmodel.CollectorConnections, hostname string) {
-			var count *countCnx
-			var found bool
-			if count, found = countConnections[hostname]; !found {
-				countConnections[hostname] = &countCnx{}
-				count = countConnections[hostname]
-			}
-			count.hit++
-
-			switch c.Type {
-			case agentmodel.ConnectionType_tcp:
-				count.TCP++
-			case agentmodel.ConnectionType_udp:
-				count.UDP++
-			}
-			validateConnection(t, c, cc, hostname)
-		})
-
-		totalConnections := countCnx{}
-		for host, count := range countConnections {
-			t.Logf("connections %d tcp %d udp %d received by host/netID %s", count.hit, count.TCP, count.UDP, host)
-			totalConnections.hit += count.hit
-			totalConnections.TCP += count.TCP
-			totalConnections.UDP += count.UDP
-		}
-		t.Logf("sum connections %d tcp %d udp %d", totalConnections.hit, totalConnections.TCP, totalConnections.UDP)
-	}, 60*time.Second, time.Second)
+	vt.Run("host_requests", func(t *testing.T) {
+		v.BaseSuite.SetT(t)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Seeing this, I wonder if this should be a suite subtest instead

Reply:
on the 2nd trough I think it would make sense to go this way and avoid the .SetT(t) and .before() calls